tests/integration-tests/tests/update/test_update.py

```diff
@@ -110,16 +162,43 @@ def _test_update_compute_instance_type(region, cluster, new_compute_instance):
         max_monitoring_time=minutes(estimated_scaleup_time),
         number_of_nodes=number_of_nodes,
     )
-    _test_compute_instance_type(region, cluster.cfn_name, new_compute_instance)
+
+    return [node for node in slurm_commands.get_compute_nodes() if node not in initial_compute_nodes]
 
 
-def _test_compute_instance_type(region, stack_name, compute_instance_type):
+def _test_compute_instance_type(region, stack_name, compute_instance_type, host, must_fail=False):
+    hostname = "{0}.{1}.compute.internal".format(host, region)
     ec2_resource = boto3.resource("ec2", region_name=region)
     instance_types = []
-    for instance in ec2_resource.instances.filter(Filters=[{"Name": "tag:Application", "Values": [stack_name]}]):
+    for instance in ec2_resource.instances.filter(
+        Filters=[
+            {"Name": "tag:Application", "Values": [stack_name]}, {"Name": "private-dns-name", "Values": [hostname]}
+        ]
+    ):
         instance_types.append(instance.instance_type)
 
-    assert_that(instance_types).contains(compute_instance_type)
+    if must_fail:
+        assert_that(instance_types).does_not_contain(compute_instance_type)
+    else:
+        assert_that(instance_types).contains(compute_instance_type)
+
+
+def _test_compute_root_volume_size(
+        command_executor, slurm_commands, test_datadir, compute_root_volume_size, host, must_fail=False
+):
+    # submit a job to retrieve compute root volume size and save in a file
+    result = slurm_commands.submit_script(str(test_datadir / "slurm_get_root_volume_size.sh"), host=host)
```

Comment:
Did you forget to add the script to the PR?

Reply:
Added